src/ontology/mondo-edit.obo

```diff
@@ -535301,6 +535308,7 @@ is_a: MONDO:0030309 {source="OMIM:620569"} ! Leber hereditary optic neuropathy,
 [Term]
 id: MONDO:0958199
 name: myoclonic epilepsy of Lafora 1
+def: "Any Lafora disease in which the cause of the disease is a variation in the EPM2A gene." [MONDO:patterns/disease_series_by_gene]
```

Comment:
Should this term also have the `has material basis in germline mutation in` property?

Reply:
added